packages/perseus-core/src/widgets/sorter/sorter-util.ts

```diff
@@ -17,13 +21,22 @@ type SorterPublicWidgetOptions = {
 function getSorterPublicWidgetOptions(
     options: PerseusSorterWidgetOptions,
 ): SorterPublicWidgetOptions {
+    const shuffledCorrect = CoreUtil.shuffle(
+        options.correct,
+        Math.random() * 100,
```

Comment:
Why multiply by 100 here?

I think you can just do

```suggestion
        Math.random,
```

to use `Math.random` as the random number generator for shuffling.

Reply:
This is based off of Matcher's PR, so there's probably more details there, but I mentioned in sync that I couldn't get the order to randomize on every test run unless I multiplied by 100. But that was with the parenthesis. Maybe it's different if I just pass Math.random. Will test!

Reply:
The reason multiplying by 100 makes it random is that `seededRNG` uses bitwise operations like `<< n` (left shift by n bits) and `& n` (bitwise and with n) which convert the operands to integers. `Math.random()` returns floats in the range [0, 1), and when those are converted to integers, the result is always 0.

If you multiply by 100, though, you get a random number in the range [0, 100), so there are 100 distinct seeds.

Even then, though, the results of shuffling won't be very random, because 100 different seeds is not that many. There are over 3 million possible permutations of a 10-element array, and if you only have 100 different seeds for your RNG, only 100 of those possible permutations will ever be selected.

This might not seem like that big a deal, but _it leaks information to cheaters_ because they can then infer the original order of the array from the shuffled order with better-than-chance accuracy.

In summary, I think passing `Math.random` as the RNG function, like I suggested above, is the way to go.

Reply:
Thanks so much for pointing this out! Will update in Matcher too